### ISSUE TITLE
Compress images

### ISSUE THREAD

**Issue by unclesam2024** [web-infrastructure/doc_architecture#42]
Images should be compressed so they load quickly.

Often websites encounter this problem when large images are added via their CMS. Unless the CMS automatically optimizes images for you, a large upload can result in an extraordinary amount of data for your visitors to download, especially on mobile devices.

# How this check works
Every JPEG and BMP image on the page is checked in turn.

For each image, Silktide compares against the same image with a compression level of 85. If the potential savings are 4Kb or greater, the image is marked for potential optimization.

# How to fix
Most popular CMS' provide support for automatically optimizing your images.

You may also wish to consider using a CDN to serve your images, which can help with this and many related performance problems.

# Further reading
[Efficiently encoding images](https://web.dev/uses-optimized-images/) by Google.
